.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,31 @@
+name: deploy-test
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+    - main
+
+jobs:
+  deploy-basic:
+    runs-on: ubuntu-latest
+    steps:
+
+    - uses: actions/checkout@v3
+
+    - uses: actions/setup-go@v4
+      with:
+        go-version-file: go.mod
+
+    - name: Run basic unit tests
+      run: |
+        make kind-cluster
+        make install
+        kubectl get crds operators.operators.operatorframework.io
+        make uninstall
+        ! kubectl get crds operators.operators.operatorframework.io
+        make deploy
+        kubectl get ns operator-controller-system
+        make undeploy
+        ! kubectl get ns operator-controller-system
```

Comment:
These steps essentially happen during the course of running the e2e, don't they? I suppose we don't try `uninstall` and `undeploy` during the e2e, but maybe we could?